Makefile

```diff
@@ -151,10 +171,12 @@ $(pkg_targets): package/%:
 	$(MAKE) yamlfile=$(yamlfile) pkgname=$* packages/$(ARCH)/$(pkgver).apk
 
 packages/$(ARCH)/%.apk: cache $(KEY) $(QEMU_KERNEL_DEP)
+	$(MAKE) cache-tokens-if-needed/$(yamlfile)
 	mkdir -p ./$(pkgname)/
 	$(eval SOURCE_DATE_EPOCH ?= $(shell git log -1 --pretty=%ct --follow $(yamlfile)))
 	$(info @SOURCE_DATE_EPOCH=$(SOURCE_DATE_EPOCH) $(MELANGE) build $(yamlfile) $(MELANGE_OPTS) --source-dir ./$(pkgname)/)
 	SOURCE_DATE_EPOCH=$(SOURCE_DATE_EPOCH) $(MELANGE) build $(yamlfile) $(MELANGE_OPTS) --source-dir ./$(pkgname)/
+	$(MAKE) clean-tokens
```

Comment:
I think this would be racy if someone did a `make -j3 package/foo package/bar package/baz` - that is, the token would be changing or getting deleted out from under the other builds.

This makes me want to go back to the idea of putting the token in the package build directory instead of the cache. Each package would then have its own tokens. They could live for just that package build, and other builds that happen to run at the same time wouldn't get a copy of the tokens from another build. WDYT?

Reply:
Didn't catch this, but you're right! We should do the same for guarded repos and libraries. I'll make the change